controllers/storagecluster/cephcluster.go

```diff
@@ -766,6 +767,35 @@ func getMonCount(sc *ocsv1.StorageCluster) int {
 	return defaults.DefaultMonCount
 }
 
+// getFullRatios returns the ratio for the given ratio name
+// If the ratio is not set in the StorageCluster CR, it returns the default value
+func getFullRatios(sc *ocsv1.StorageCluster, ratioName string) *float64 {
+	mCCSpec := sc.Spec.ManagedResources.CephCluster
+
+	switch ratioName {
+	case "nearFull":
+		if mCCSpec.NearFullRatio != nil {
+			return mCCSpec.NearFullRatio
+		}
+		return ptr.To(defaults.DefaultNearFullRatio)
```

Comment:
Based on the above discussion, we would want to return null from this method instead of the default values
```suggestion
		return null
```